code/game/objects/items/storage/backpack.dm

```diff
@@ -420,14 +419,33 @@
 	has_gamemode_skin = FALSE //same sprite for all gamemode.
 	var/camo_active = 0
 	var/camo_active_timer = 0
-	var/camo_cooldown_timer = 0
-	var/camo_ready = 1
+	var/camo_cooldown_timer = null
+	var/camo_last_stealth = null
+	var/camo_last_shimmer = null
+	var/camo_energy = 100
+	var/mob/living/wearer = null
```

Comment:
unless `/mob/living/silicon`, `/mob/living/carbon/monkey`, etc are going to be using this you should use `var/mob/living/carbon/human/wearer`
Synths are `/mob/living/carbon/human`